android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/splash/SplashScreen.kt

```diff
@@ -0,0 +1,44 @@
+package co.nimblehq.ic.kmm.suv.android.ui.screens.splash
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.ic.kmm.suv.android.R
+import kotlinx.coroutines.delay
+
+@Composable
+fun SplashScreen(onDidLaunch: () -> Unit) {
+    LaunchedEffect(key1 = true) {
+        delay(3000L)
+        onDidLaunch()
+    }
+    Box {
+        Box(modifier = Modifier.fillMaxSize()) {
+            Image(
+                painter = painterResource(id = R.drawable.bg_splash),
+                contentDescription = null,
+                contentScale = ContentScale.FillWidth,
+                modifier = Modifier
+                    .matchParentSize()
+            )
+        }
+        Image(
+            painter = painterResource(id = R.drawable.ic_logo_white),
+            contentDescription = null,
+            modifier = Modifier.align(Alignment.Center)
+        )
```

Comment:
With the convention, we should define the `modifier` on top of params.

Reply:
@hoangnguyen92dn For `modifier`, we are using it for the `Image`/`Text` components, not define a new Compose UI, so I think we don't need to move `modifier` to the top, for example:

```kotlin
Image(
	modifier = Modifier.align(Alignment.Center),
	painter = painterResource(id = R.drawable.ic_logo_white),
	contentDescription = null
)

or
Text(
	modifier = Modifier.align(Alignment.Center),
	text = Greeting().greeting()
)
```

The examples above look not right, right? Or I'm missing sth?

Reply:
@suho 🤝